### PR TITLE
well documented CCNode member functions and variables

### PR DESCRIPTION
**Pull request by walzer**

### CODE REVIEW

File: cocos2dx/base_nodes/CCNode.cpp [hunk 1]
```diff
@@ -282,37 +282,12 @@ void CCNode::setPosition(const CCPoint& newPosition)
...
-
-void CCNode::setPositionX(float x)
-{
-    setPosition(ccp(x, m_obPosition.y));
-}
-
-void CCNode::setPositionY(float y)
-{
-    setPosition(ccp(m_obPosition.x, y));
-}
```
Comment (dumganhar) [inline cocos2dx/base_nodes/CCNode.cpp:314]: Deleted?

File: cocos2dx/base_nodes/CCNode.cpp [hunk 2]
```diff
@@ -282,37 +282,12 @@ void CCNode::setPosition(const CCPoint& newPosition)
     m_bTransformDirty = m_bInverseDirty = true;
 }
 
-const CCPoint& CCNode::getPositionLua(void)
-{
-    return m_obPosition;
-}
```
Comment (dumganhar) [inline cocos2dx/base_nodes/CCNode.cpp:288]: Deleted? It's needed for lua.

Comment (walzer) [inline cocos2dx/base_nodes/CCNode.cpp:288]: This function is totally the same as const CCPoint& getPosition(void). Please bind getPosition() instead,

**Comment by walzer** [comment]
Add [set|get]Position[X|Y](void) methods back to CCNode, although this is ugly.

File: cocos2dx/base_nodes/CCNode.h [hunk 3]
```diff
@@ -123,619 +123,1215 @@ enum {
...
-    CCScheduler *m_pScheduler;
+    /**
+     * Default constructor
+     */
+    CCNode(void);
     
-    // ActionManager used to handle all the actions
-    CCActionManager *m_pActionManager;
+    /**
+     * Default distructor
```
Comment (dumganhar) [inline cocos2dx/base_nodes/CCNode.h:137]: distructor -> destructor

File: cocos2dx/base_nodes/CCNode.h [hunk 4]
```diff
@@ -123,619 +123,1215 @@ enum {
...
-
-public:
-    // getter & setter
+    ///////////////////////////////////////////////////////
+    /// @{
+    /// @name Setters & Getters for Graphic Peroperties
     
-    /** The z order of the node relative to it's "brothers": children of the same parent */
+    /**
+     * Sets the Z order which stand for the drawing order, and reorder this node in its parent's children array.
```
Comment (dumganhar) [inline cocos2dx/base_nodes/CCNode.h:162]: stand -> stands?

**Pull request merged**